Makefile

```diff
@@ -2,7 +2,6 @@ DESTDIR?=
 USR_DIR?=/usr/local
 INSTALL_DIR?=${DESTDIR}${USR_DIR}
 INSTALL_BIN_DIR?=${INSTALL_DIR}/bin
-GOPATH?=$(shell go env GOPATH)
```

Comment:
There're also some other blocks using this environment, we can find more GOPATH related docs or codes by searching `$GOPATH` or `GOPATH` in kubeedge files.

Reply:
OK, just have a double check, for shell script, GOPATH is a env value, it will exists once golang installed. Let's keep the env check, and just remove in Makefile.